### PR TITLE
exercise on 27 and update some privious exercise

### PR DESCRIPTION
**Pull request by bikashstackmybiz**
exercise on 27 and update some privious exercise please review it

### CODE REVIEW

File: StaticMethod.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+public class StaticMethod {
+    int num1 = 3;
+    public int num2 = 5;
+    static int num3 = 6;
+
+    static void method1(){
+       // System.out.println("this is example of static method"+num1);
+       // System.out.println("this is example of static method"+num2);  ( both are wrong beacouse we have to use static variable in static  method)
+
+        System.out.println("this is example of static method"+num3);
```
Comment (rakesh-stackmybiz) [inline StaticMethod.java:10]:
```suggestion
        System.out.println("this is example of static method" + num3);
```

File: StaticMethod.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
+public class StaticMethod {
+    int num1 = 3;
+    public int num2 = 5;
+    static int num3 = 6;
+
+    static void method1(){
```
Comment (rakesh-stackmybiz) [inline StaticMethod.java:6]: provide some proper name to this method

File: StaticVariableExample.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
+public class StaticVariableExample {
+    private static int number = 0;
+
+     StaticVariableExample (int a){
+    this.number = a;
+    }
+
+    public void method1 () {
```
Comment (rakesh-stackmybiz) [inline StaticVariableExample.java:8]: make a habit of using proper name for any method.

**Comment by bikashstackmybiz** [comment]
done as per as suggestion

**Comment by bikashstackmybiz** [comment]
done as per as  suggestion

**Comment by bikashstackmybiz** [comment]
updated as per as suggestion

**Pull request merged**